File: app/src/dev/java/com/njxm/smart/GlobalConst.java
package com.njxm.smart;

/**
 * 测试环境全局变量
 */
public class GlobalConst {
    private static final String HTTP_PROTOCOL = "http://";


    public static final String PRD_IP = "192.168.3.112";

    /**
     * H5 端口号
     */
    public static final String H5_PORT = "8080";

    /**
     * 业务端口号
     */
    public static final String BIZ_PORT = "8083";

    /**
     * 校验端口号
     */
    public static final String GRANT_PORT = "8081";


    public static final String URL_GRANT_PREFIX = HTTP_PROTOCOL + PRD_IP + ";" + GRANT_PORT;
    public static final String URL_BIZ_PREFIX = HTTP_PROTOCOL + PRD_IP + ":" + BIZ_PORT;
    public static final String URL_H5_PREFIX = HTTP_PROTOCOL + PRD_IP + ":" + H5_PORT;
}

File: app/src/main/java/com/njxm/smart/GlobalConst.kt
package com.njxm.smart

import java.text.MessageFormat

/**
 * 测试环境全局变量
 */
object GlobalConst {
    /**
     * Http 协议
     */
    private const val HTTP_PROTOCOL = "http"

    /**
     * Url格式
     */
    private const val HOST_MESSAGE = "{1}://{2}:{3}"

    /**
     * H5 URL
     */
    val h5UrlPrefix: String
        get() = MessageFormat.format(HOST_MESSAGE, HTTP_PROTOCOL, HttpConstant.HOST, HttpConstant.PORT_H5)

    /**
     * Biz URL
     */
    val bizUrl: String
        get() = MessageFormat.format(HOST_MESSAGE, HTTP_PROTOCOL, HttpConstant.BIZ_PORT)

    /**
     * VER URL
     */
    val keyVerify: String
        get() = MessageFormat.format(HOST_MESSAGE, HTTP_PROTOCOL, HttpConstant.VER_PORT)
}